scripts/injectLicense.js

```diff
@@ -36,7 +36,9 @@ files.forEach(file => {
     }
 
     if (!data.includes(license)) {
-      prependFile(file, license, err => {
+      const pre = data.includes('@flow') ? license : license + '/* @flow */\n';
```

Comment:
Now that this files handles more than licensing, maybe change the name to format.js?